x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -292,6 +297,7 @@ export interface YConfig {
   forAccessor: string;
   axisMode?: YAxisMode;
   color?: string;
+  showValueLabels?: boolean;
```

Comment:
Based on my review of the rest of this code, I think we should discuss how to represent this in the XY state. My preference here is to say that there is a "value label config" which is attached to the overall XY state, outside of any specific layer. I'm imagining properties like this:

```typescript
interface ValueLabelConfig {
  showLabels: boolean;
  fontSize?: number;
  position?: 'inside' | 'outside';
}
```

which would include the most common options we might want to provide users